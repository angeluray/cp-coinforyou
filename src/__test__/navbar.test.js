import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Navigation from '../components/navbar';

it('Test navbar renders correctly', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Navigation />
      </BrowserRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
