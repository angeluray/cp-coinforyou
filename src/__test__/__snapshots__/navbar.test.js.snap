// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Navbar renders correctly 1`] = `
<nav>
  <a
    aria-current="page"
    className="navlink active"
    href="/"
    onClick={[Function]}
  >
    <img
      alt="Back icon"
      className="back-icon"
      src="back.png"
    />
    <p>
      BACK
    </p>
  </a>
  <a
    aria-current="page"
    className="navlink-1 active"
    href="/"
    onClick={[Function]}
  >
    <img
      alt="CryptoForU Logo"
      className="logo"
      src="navlogo-short.png"
    />
  </a>
  <ul
    className="nav-ul"
  >
    <li>
      <img
        alt="Microphone icon"
        className="nav-options"
        src="microphone.png"
      />
    </li>
    <li>
      <img
        alt="Settings icon"
        className="nav-options"
        src="settings.png"
      />
    </li>
  </ul>
</nav>
`;

exports[`Test navbar renders correctly 1`] = `
<nav>
  <a
    aria-current="page"
    className="navlink active"
    href="/"
    onClick={[Function]}
  >
    <img
      alt="Back icon"
      className="back-icon"
      src="back.png"
    />
    <p>
      BACK
    </p>
  </a>
  <a
    aria-current="page"
    className="navlink-1 active"
    href="/"
    onClick={[Function]}
  >
    <img
      alt="CryptoForU Logo"
      className="logo"
      src="navlogo-short.png"
    />
  </a>
  <ul
    className="nav-ul"
  >
    <li>
      <img
        alt="Microphone icon"
        className="nav-options"
        src="microphone.png"
      />
    </li>
    <li>
      <img
        alt="Settings icon"
        className="nav-options"
        src="settings.png"
      />
    </li>
  </ul>
</nav>
`;
